processor/resourcedetectionprocessor/internal/aws/ecs/ecs.go

```diff
@@ -92,10 +91,10 @@ func (d *Detector) Detect(context.Context) (pdata.Resource, error) {
 	// The launch type and log data attributes are only available in TMDE v4
 	switch lt := strings.ToLower(tmdeResp.LaunchType); lt {
 	case "ec2":
-		attr.InsertString("aws.ecs.launchtype", "EC2")
+		attr.InsertString("aws.ecs.launchtype", "ec2")
```

Comment:
If it's possible, it'd generally be better to have separate PRs for this processor and the exporter

Reply:
Sorry, definitely makes sense. In this case I realized some issues with the detector as I was making the exporter changes, and made them side-by-side. In the future I will definitely take care to separate PRs by module.